alws/dramatiq/tests.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import typing
+
+import dramatiq
+from sqlalchemy import update
+
+from alws.constants import DRAMATIQ_TASK_TIMEOUT, TestTaskStatus
+from alws.crud import test as t_crud
+from alws.database import Session
+from alws.dramatiq import event_loop
+from alws.models import TestTask
+from alws.schemas.test_schema import TestTaskResult
+
+
+__all__ = ['complete_test_task']
+
+
+async def _complete_test_task(task_id: int, task_result: TestTaskResult):
+    async with Session() as db, db.begin():
+        try:
+            await t_crud.complete_test_task(db, task_id, task_result)
+        except Exception as e:
+            logging.exception(
+                'Cannot set test task "%d" result, marking as failed.'
+                'Error: %s', task_id, str(e))
+            await db.rollback()
+        await db.execute(update(TestTask).where(
+            TestTask.id == task_id).values(status=TestTaskStatus.FAILED))
+
+
+@dramatiq.actor(max_retries=2, priority=1, time_limit=DRAMATIQ_TASK_TIMEOUT)
```

Comment:
Please, set a priority for higher value, sign tasks and build done also have priority=1, this will make impossible for build to complete or sign if test system is under load

Reply:
Fixed